internal/pkg/template/repository/manifest/records.go

```diff
@@ -75,6 +75,31 @@ func (v *TemplateRecord) GetVersion(wanted model.SemVersion) (VersionRecord, boo
 	return VersionRecord{}, false
 }
 
+func (v *TemplateRecord) GetVersionOrErr(wantedStr string) (VersionRecord, error) {
+	// Parse version
+	var wanted model.SemVersion
+	if wantedStr == "" {
+		if v, err := v.DefaultVersionOrErr(); err != nil {
+			return VersionRecord{}, err
+		} else {
+			wanted = v.Version
+		}
+	} else {
+		if v, err := model.NewSemVersion(wantedStr); err != nil {
+			return VersionRecord{}, err
+		} else {
+			wanted = v
+		}
+	}
```

Comment:
A sa zada `wantedStr = ""`, tak sa pouzije default verzia.
To sa potom vyuziva na viacerych miesta v kode, a odstranil sa tak duplicitny kod.